docs/algos/performances.md

```diff
@@ -0,0 +1,43 @@
+# Performance assessments
+
+:warning: This document is a work in progress.
+
+## Introduction
+To ensure the implementation of the algorithms are correct, we want to test them on various environments. For the sake of reproducibility, we want to run for 10 seeds on various environments. For maintenance purposes and long-term support, these tests will be conducted on environments available in [MO-Gymnasium](www.github.com/farama-foundation/mo-gymnasium). Hence, we will not be able to test on some environments that were presented in original papers.
+
+
+## Metrics
+For single-policy algorithms, the metric used will be the scalarized return of the policy.
```

Comment:
The single-policy algorithms we have are all for non-linear utilities, right? In that case, I believe you meant to use the utility as the metric?